### ISSUE TITLE
Problems with :wqa

### ISSUE THREAD

**Issue by felipejoribeiro** [kevinhwang91/rnvimr#76]
- `nvim --version`: NVIM v0.5.0
- Operating system/version: 5.10.19-1-lts
- `nvim +'checkhealth rnvimr'`: 
```
health#rnvimr#check
========================================================================
## OS
  - OK: Name: Linux

## Ranger
  - OK: Version: ranger 1.9.3

## Python
  - OK: Version: 3.9.2 (default, Feb 20 2021, 18:40:11) [GCC 10.2.0]

## Pynvim
  - OK: Version: 0.4.2

## Ueberzug (optional)
  - OK: Ueberzug is ready

## RPC
  - OK: RPC echo: Neovim send "Give me five!" and receive "Give me five!"

```
**Describe the bug**
When the ranger file explorer is used in a vim session, it becomes impossible to use the command `:wqa`.

**To Reproduce using `nvim -u mini.vim`**

Example:
`cat mini.vim`
```vim
" use your plugin manager, here is `vim-plug`
call plug#begin('~/.config/nvim/plugged')
Plug 'kevinhwang91/rnvimr'
let g:rnvimr_vanilla = 1
tnoremap <silent> <M-i> <C-\><C-n>:RnvimrResize<CR>
nnoremap <silent> <M-o> :RnvimrToggle<CR>
tnoremap <silent> <M-o> <C-\><C-n>:RnvimrToggle<CR>
call plug#end()
```

1. Just fire the ranger instance with `<M-o>`.
2. Close the ranger instance with `q`.
3. Try to quit nvim with the command `:wqa`

**Expected behavior**
Leave vim with alteration in all buffers saved as usual.

**Screenshots**
Follows the error menssage when trying to quit the program with `:wqa` :

![image](https://user-images.githubusercontent.com/39244876/110060029-06c8c080-7d44-11eb-9040-ead4af539e05.png)

**Additional context**
The error in plain text:
```
E948: Job still running
E676: No matching autocommands for acwrite buffer
Press ENTER or type command to continue
```
Thank you for the awesome plugin. Hope this info may help somehow

**Comment by kevinhwang91** [comment]
Thanks for your feedback, I can reproduce the issue in nvim-0.5 but not in the stable version. I think it's the upstream's bug.

You can also reproduce it by `nvim +'terminal' +'new' +'wqa'`. I will report this issue to upstream latter:)

**Comment by felipejoribeiro** [comment]
I see. Well noted! Thank you. For now i will save each buffer individually and quit all with just `:qa` as it still works.

**Comment by Rimann91** [comment]
Just wanted to report that this is still an issue

**Comment by ls-devs** [comment]
Yep, still an issue even today, should we wait for neovim to patch this or can we do something ?

**Comment by Penaz91** [comment]
I ran into the same issue using `:xa`, I mitigated the issue with the following line added in my init.lua:

```
vim.cmd[[cabbrev xa wa\|qa]]
```

It is extremely crude and I don't know which kind of side effects it could have, but I thought this could be a workaround of sorts.
